### PR TITLE
[Claudia]Feature file with 3 scenarios

### PR DESCRIPTION
**Pull request by Claudia-20**
Claudia feature file

### CODE REVIEW

File: src/test/resources/feature/TestScenarios.feature [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@Claudia
+Feature: Add, remove and modify items from shopping cart
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/TestScenarios.feature]: Could you please rename your TestScenarios.feature file to more specific (the area you are testing).

File: src/test/resources/feature/TestScenarios.feature [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  Scenario: Add items to cart
+
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/TestScenarios.feature]: Please remove empty line after Scenario all over the file.

File: src/test/resources/feature/TestScenarios.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Given user is on items page
+    When user adds items with following price
+      | Item price |
+      | $16.51      |
+      | $27.00      |
+    And click on [Add to cart] button
+    Then a pop up window is displayed with message "Product successfully added to your shopping cart"
+    And item details are displayed
+
+  Scenario: Remove items from cart
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/TestScenarios.feature]: If you are starting here (and in the Modify number of items from shopping cart scenario) also from items page (you have added this step in previous one scenario "Given user is on items page") please add this step in Background for all of the scenarios.

File: src/test/resources/feature/TestScenarios.feature (hunk 1 above)
Comment (Claudia-20) [reply src/test/resources/feature/TestScenarios.feature]: Done

File: src/test/resources/feature/TestScenarios.feature (hunk 2 above)
Comment (Claudia-20) [reply src/test/resources/feature/TestScenarios.feature]: Done

File: src/test/resources/feature/TestScenarios.feature (hunk 3 above)
Comment (Claudia-20) [reply src/test/resources/feature/TestScenarios.feature]: Done

**Review by VitalinaPetrenco — APPROVED** [review]

**Pull request merged**